packages/provider-url-service/src/index.ts

```diff
@@ -77,5 +95,16 @@ export const composeCicoProviderUrl = functions.https.onRequest((request, respon
       `.replace(/\s+/g, '')
   }
 
-  response.send(JSON.stringify(finalUrl))
-})
+  return finalUrl
+}
+
+const composeIpUrl = (requestData: IpRequestData) => {
+  const { apiUrl, public_key } = MOONPAY_DATA[requestData.env]
+
+  const url = `
+    ${apiUrl}/v4/ip_address
+        ?apiKey=${public_key}
+      `.replace(/\s+/g, '')
+
+  return url
+}
```

Comment:
Is this worth a fetch request to this service to assemble the final url given there's nothing private required to do it?

I think one concern was about being able to change this dynamically, but soon we'll have a generic solution for remote configuration where a bunch of flags/config will be move to it.

Reply:
I'd rather not add an additional network request if it can be avoided. The primary concern was the future potential need for key rotation or a change in the API version, which would be breaking changes. If we have a generalized solution for this in the works, I'm happy to have this client-side for now.